i18n/es-AR/messages.po

```diff
@@ -0,0 +1,7 @@
+msgid ""
```

Comment:
we don't version the `po` files, this are created by gettext

Reply:
we gitignore the i18n folder

Reply:
yeah, I know, but this project does not exist in Crowdin so you wouldn't have any key to test the results. Remember this is a PoC in a disposable repo 🙃